projects/nfdi-gdo-app/src/app/species-selector/species-selector.component.ts

```diff
@@ -7,9 +10,160 @@ import {ChangeDetectionStrategy, Component, OnDestroy, OnInit} from '@angular/co
-    constructor() {}
+    datasetId: UUID = 'd9dd4530-7a57-44da-a650-ce7d81dcc216';
+
+    species: string[] = [
+        'Aeshna affinis',
+        'Aeshna caerulea',
+        'Aeshna cyanea',
+        'Aeshna grandis',
+        'Aeshna isoceles',
+        'Aeshna juncea',
+        'Aeshna mixta',
+        'Aeshna subarctica',
+        'Aeshna viridis',
+        'Anax ephippiger',
+        'Anax imperator',
+        'Anax parthenope',
+        'Boyeria irene',
+        'Brachytron pratense',
+        'Calopteryx splendens',
+        'Calopteryx virgo',
+        'Ceriagrion tenellum',
+        'Chalcolestes viridis',
+        'Coenagrion armatum',
+        'Coenagrion hastulatum',
+        'Coenagrion lunulatum',
+        'Coenagrion mercuriale',
+        'Coenagrion ornatum',
+        'Coenagrion puella',
+        'Coenagrion pulchellum',
+        'Coenagrion scitulum',
+        'Cordulegaster bidentata',
+        'Cordulegaster boltonii',
+        'Cordulia aenea',
+        'Crocothemis erythraea',
+        'Enallagma cyathigerum',
+        'Epitheca bimaculata',
+        'Erythromma lindenii',
+        'Erythromma najas',
+        'Erythromma viridulum',
+        'Gomphus flavipes',
+        'Gomphus pulchellus',
+        'Gomphus simillimus',
+        'Gomphus vulgatissimus',
+        'Ischnura elegans',
+        'Ischnura pumilio',
+        'Lestes barbarus',
+        'Lestes dryas',
+        'Lestes sponsa',
+        'Lestes virens',
+        'Leucorrhinia albifrons',
+        'Leucorrhinia caudalis',
+        'Leucorrhinia dubia',
+        'Leucorrhinia pectoralis',
+        'Leucorrhinia rubicunda',
+        'Libellula depressa',
+        'Libellula fulva',
+        'Libellula quadrimaculata',
+        'Nehalennia speciosa',
+        'Onychogomphus forcipatus',
+        'Onychogomphus uncatus',
+        'Ophiogomphus cecilia',
+        'Orthetrum albistylum',
+        'Orthetrum brunneum',
+        'Orthetrum cancellatum',
+        'Orthetrum coerulescens',
+        'Oxygastra curtisii',
+        'Platycnemis pennipes',
+        'Pyrrhosoma nymphula',
+        'Somatochlora alpestris',
+        'Somatochlora arctica',
+        'Somatochlora flavomaculata',
+        'Somatochlora metallica',
+        'Sympecma fusca',
+        'Sympecma paedisca',
+        'Sympetrum danae',
+        'Sympetrum depressiusculum',
+        'Sympetrum flaveolum',
+        'Sympetrum fonscolombii',
+        'Sympetrum meridionale',
+        'Sympetrum pedemontanum',
+        'Sympetrum sanguineum',
+        'Sympetrum striolatum',
+        'Sympetrum vulgatum',
+    ];
+
+    constructor(private projectService: ProjectService, private dataSelectionService: DataSelectionService) {}
 
     ngOnInit(): void {}
 
     ngOnDestroy(): void {}
+
+    selectSpecies(species: string): void {
+        this.projectService.clearLayers();
+
+        const rangeStart = species;
+        const rangeStop = species + 'z';
```

Comment:
Doesn't this work for Equality?

Reply:
585b1f88f05b884bda96284c01168f2a26712dd9